internal/tools/aws/cluster_management.go

```diff
@@ -202,6 +204,35 @@ func (a *Client) GetAndClaimVpcResources(clusterID, owner string, logger log.Fie
 	return ClusterResources{}, fmt.Errorf("%d VPCs were returned as currently available; none of them were configured correctly", len(vpcs))
 }
 
+// GetVpcResources retrieve the VPC information for a particulary cluster.
+func (a *Client) GetVpcResources(clusterID string, logger log.FieldLogger) (ClusterResources, error) {
+	// First, check if a VPC has been claimed by this cluster. If only one has
+	// already been claimed, then return that with no error.
+	clusterAlreadyClaimedFilter := []*ec2.Filter{
+		{
+			Name: aws.String(VpcAvailableTagKey),
+			Values: []*string{
+				aws.String(VpcAvailableTagValueFalse),
+			},
+		},
+		{
+			Name: aws.String(VpcClusterIDTagKey),
+			Values: []*string{
+				aws.String(clusterID),
+			},
+		},
+	}
+	clusterAlreadyClaimedVpcs, err := a.GetVpcsWithFilters(clusterAlreadyClaimedFilter)
+	if err != nil {
+		return ClusterResources{}, err
+	}
+	if len(clusterAlreadyClaimedVpcs) == 0 {
```

Comment:
We may want to check for `len(clusterAlreadyClaimedVpcs) != 1` in case two VPCs were somehow returned. It's probably safe enough to not check this, but just wanted to note it.

Reply:
done